eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
-# Ballerina Azure Event Hubs Module
+## Overview
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
+Ballerina Connector for Azure Event Hub provides access to the Azure Event Hub via Ballerina language to ingest, buffer, store, and process high-frequency data from any source in real-time to derive important business insights. 
+It provides the capability to perform Event Hub service operations, Event Hub management operations and the capability to handle publisher policy operations.
 
-# Module Overview
+This module supports [Azure Event Hub REST API](https://docs.microsoft.com/en-us/rest/api/eventhub/) 2014-01 version.
 
-Azure Event Hubs Ballerina Connector is used to connect with the Azure Event Hubs to ingest millions of events per 
-second so that you can process and analyze the massive amounts of data produced by your connected devices and 
-applications. Once data is collected into an Event Hub, it can be transformed and stored using any real-time 
-analytics provider or batching/storage adapters.
+## Prerequisites
 
-Azure Event Hub Ballerina connector supports Event hub service operations like sending an event, sending batch events, 
-sending partition events and sending events with partition ID. It also supports Event hub management operations like 
-creating a new event hub, getting an event hub, updating an event hub, listing event hubs, deleting event hubs, 
-creating a new consumer group, getting consumer groups, listing consumer groups, listing partitions, getting partitions, 
-deleting consumer groups. The connector also provides the capability to handle publisher policy operations like getting 
-revoked publishers, revoking a publisher, and resume publishers.
+Before using this connector in your Ballerina application, complete the following:
 
-The connector will only be focusing on sending events to the event hub. The event hub connector will invoke the 
-REST APIs exposed via the Azure Event Hub. https://docs.microsoft.com/en-us/rest/api/eventhub/.
+* [Create Azure account to access Azure portal](https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/)
 
-The REST APIs fall into the following categories:
-* Azure Resource Manager: APIs that perform resource manager operations, and have /providers/Microsoft.EventHub/ as part 
-  of the request URI.
-* Event Hubs service: APIs that enable operations directly on the Event Hubs service, and have 
-  <namespace>.servicebus.windows.net/ in the request URI. The Event Hubs service API is focused on this implementation.
+* [Create resource group](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-a-resource-group)
 
-# Compatibility
-|                     |    Version          |
-|:-------------------:|:-------------------:|
-| Ballerina Language  | Swan-Lake-Alpha5    |
+* [Create Event Hubs namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hubs-namespace)
```

Comment:
Create a Event hubs namespace

Reply:
Resolved and amended the changes. Thank you for the suggestion.